<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(133, 150, 105);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_logouat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Imagenes/recursos1.qrc">:/logos/LogoUAT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_logofit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Imagenes/recursos1.qrc">:/logos/LogoFIT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_logoequipo">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Imagenes/recursos1.qrc">:/logos/LogoEquipo.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_persona">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Imagenes/recursos1.qrc">:/logos/Angel.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Equipo:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_angel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Angel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_jorge">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Jorge</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_persona_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Imagenes/recursos1.qrc">:/logos/Jorge.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_persona_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Imagenes/recursos1.qrc">:/logos/Paniagua.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_paniagua">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Juan</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_sheko">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sergio</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_persona_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Imagenes/recursos1.qrc">:/logos/Sheko.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_equipo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Imagenes/recursos1.qrc"/>
 </resources>
 <connections/>
</ui>
